File: 1.py
arr = [3,234,34,5,32,199,2999,4,54]
n = len(arr)
for i in range(n-1):
    for j in range(n-i-1):
        if ( arr[j] > arr[j+1]):
            temp = arr[j]
            arr[j] = arr[j+1]
            arr[j+1] = temp
print(arr)

File: 1.cpp
#include <iostream>

int main() {
    int arr[] = {0,9,999,99,88,77,444,666,111,1000};
    int n = sizeof(arr) / sizeof(arr[0]);
    
    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                int temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
    
    for (int i = 0; i < n; i++) {
        std::cout << arr[i] << " ";
    }
    
    return 0;
}